go/adbc/pkg/Makefile

```diff
@@ -14,7 +14,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-MANAGERS = bigquery flightsql panicdummy snowflake
```

Comment:
Can you run `make regenerate` to make sure the C code is up to date with the latest main? I think it probably is, but always good to check.